backend_new/prisma/schema.prisma

```diff
@@ -0,0 +1,1102 @@
+model Applicant {
+  id                   String        @id() @default(dbgenerated("uuid_generate_v4()")) @db.Uuid
+  createdAt            DateTime      @default(now()) @map("created_at") @db.Timestamp(6)
+  updatedAt            DateTime      @updatedAt @map("updated_at") @db.Timestamp(6)
+  firstName            String?       @map("first_name")
+  middleName           String?       @map("middle_name")
+  lastName             String?       @map("last_name")
+  birthMonth           String?       @map("birth_month")
+  birthDay             String?       @map("birth_day")
+  birthYear            String?       @map("birth_year")
+  emailAddress         String?       @map("email_address")
+  noEmail              Boolean?      @map("no_email")
+  phoneNumber          String?       @map("phone_number")
+  phoneNumberType      String?       @map("phone_number_type")
+  noPhone              Boolean?      @map("no_phone")
+  workInRegion         YesNoEnum?    @map("work_in_region")
+  workAddressId        String?       @unique() @map("work_address_id") @db.Uuid
+  addressId            String?       @unique() @map("address_id") @db.Uuid
+  applicantWorkAddress Address?      @relation("applicant_work_address", fields: [workAddressId], references: [id], onDelete: NoAction, onUpdate: NoAction)
+  applicantAddress     Address?      @relation("applicant_address", fields: [addressId], references: [id], onDelete: NoAction, onUpdate: NoAction)
+  applications         Applications?
+
+  @@map("applicant")
+}
+
+// Note: [rule] used to be a different enum but prisma doesn't support that kind of enum yet. See: https://github.com/prisma/prisma/issues/273
+model ApplicationFlaggedSet {
+  id                    String               @id() @default(dbgenerated("uuid_generate_v4()")) @db.Uuid
+  createdAt             DateTime             @default(now()) @map("created_at") @db.Timestamp(6)
+  updatedAt             DateTime             @updatedAt @map("updated_at") @db.Timestamp(6)
+  rule                  RuleEnum
+  ruleKey               String               @unique() @map("rule_key") @db.VarChar
+  resolvedTime          DateTime?            @map("resolved_time") @db.Timestamptz(6)
+  listingId             String               @map("listing_id") @db.Uuid
+  showConfirmationAlert Boolean              @default(false) @map("show_confirmation_alert")
+  status                FlaggedSetStatusEnum @default(pending)
+  resolvingUserId       String?              @map("resolving_user_id") @db.Uuid
+  userAccounts          UserAccounts?        @relation(fields: [resolvingUserId], references: [id], onDelete: NoAction, onUpdate: NoAction)
+  listings              listings             @relation(fields: [listingId], references: [id], onDelete: NoAction, onUpdate: NoAction)
+  applications          Applications[]
+
+  @@index([listingId])
+  @@map("application_flagged_set")
+}
+
+// model application_flagged_set_applications_applications {
```

Comment:
What is the plan for these seven commented out models?
```
application_flagged_set_applications_applications
applications_preferred_unit_unit_types
jurisdictions_multiselect_questions_multiselect_questions
listings_leasing_agents_user_accounts
user_accounts_jurisdictions_jurisdictions
unit_group_unit_type_unit_types
user_preferences_favorites_listings
```

Reply:
ah dang, those aren't really "models" they were part of the old structure but we can let prisma generate those the way it needs to for its relationships.

removed em